src/index.njk

```diff
@@ -3,102 +3,172 @@ layout: default
 sitemapPriority: 1.0
 ---
 <!--Hero Content-->
-<div class="w-full px-6 pb-24 mt-12 md:px-0 ff-bg-flows">
-    <div class="container m-auto text-center max-w-2xl">
-        <h1 class="text-gray-50 max-w-xl m-auto">
-            {{ site.messaging.tagLine }}
-        </h1>
-        <p class="mt-6 text-gray-400 text-lg font-normal">
-            {{ site.messaging.subtitle }}
-        </p>
-        <div class="flex flex-col">
-            <div class="m-auto mt-7 flex flex-col gap-4 items-center justify-center">
-                <a class="ff-btn ff-btn--primary flex flex-col" href="{{ site.appURL }}/account/create" onclick="capture('cta-join', {'position': 'primary'})">
-                    <span class="text-base uppercase flex gap-2 items-center">
-                        TRY IT OUT
-                        {% include "components/icons/arrow-right.svg" %}
-                    </span>
-                </a>
+<div class="w-full px-6 pb-24 mt-12 ff-bg-flows">
+    <div class="container m-auto text-left max-w-screen-lg lg:flex lg:flex-row lg:items-start lg:justify-between lg:gap-6">
+        <div class="max-w-xl mx-auto lg:mx-0 lg:my-auto lg:py-10 lg:max-w-[600px]">
+            <h1 class="text-gray-50 max-w-xl m-auto lg:m-0 text-center lg:text-left">
+                {{ site.messaging.tagLine }}
+            </h1>
+            <p class="mt-6 text-gray-400 text-lg font-normal text-center lg:text-left">
+                {{ site.messaging.subtitle }}
+            </p>
+            <div class="flex flex-col">
+                <div
+                    class="m-auto mt-7 flex gap-4 items-center justify-center lg:items-start lg:justify-start lg:m-0 flex-row">
+                    <a class="ff-btn ff-btn--primary flex flex-col mb-6" href="{{ site.appURL }}/account/create"
+                        onclick="capture('cta-join', {'position': 'primary'})">
+                        <span class="text-base uppercase items-center">
+                            Contact us
+                        </span>
+                    </a>
+                    <a class="ff-btn flex flex-col mb-6" href="{{ site.appURL }}/account/create"
+                        onclick="capture('cta-join', {'position': 'primary'})">
+                        <span class="text-base uppercase flex gap-2 items-center text-gray-200">
+                            TRY IT OUT
```

Comment:
@Yndira-FlowForge can you please switch the button to say "Try it out" and the text to say "Contact us"

Reply:
@Hasmin-AC I don't think that's what you want, as this CTA goes to the account registration page.

Reply:
@ZJvandeWeg we discussed it over a call, she wanted the primary button to be "try it out" and the secondary "contact us", it's done 😉

Reply:
Yes, main CTA is to Sign up to the cloud and try it.